lib/cc/cli/command.rb

```diff
@@ -43,6 +43,10 @@ def require_codeclimate_yml
         end
       end
 
+      def set_local_owner
```

Comment:
What do you think about this method living on `FileSystem`?

If we eventually fix the abstraction issue I had mentioned, the calling code would look like (for example):

```rb
filesystem.write(CODECLIMATE_YAML, parsed_yaml)
filesystem.correct_permissions(CODECLIMATE_YAML)
```

And that feels nice to me.